server/src/graphql/user/queries.ts

```diff
@@ -0,0 +1,3 @@
+export const queries=`
+ getUser(email:String!,profileImageUrl:String):[User]
```

Comment:
for getUser why is profileImageUrl a parameter for search??

if i am not wrong getUser will search user with respect to email and profileImageUrl here right?